app/scripts/modules/core/src/orchestratedItem/orchestratedItem.transformer.ts

```diff
@@ -165,6 +166,29 @@ export class OrchestratedItemTransformer {
     return null;
   }
 
+  /**
+   * Get exceptions from child pipelines as references by the monitorPipeline stage
+   */
+  private static getMonitoredPipelineExceptions(task: IStage): string {
```

Comment:
I'm questioning if we really need to add another abstraction for "stage error".  Ideally errors would conform to a single model and be sourced from the same place.

Reply:
I hear you and I struggled with this myself. But I wanted to expose errors to end users (via spel so and attribute to an execution if they are monitoring multiple pipelines).. I supposed i could also aggregate it in orca for deck and have it available in both places... what do you think?

Reply:
I think this behavior is probably unique to this particular stage.  I'd prefer to see this logic isolated in the stage's execution details component.